### DISCUSSION TITLE
Expo go

### DISCUSSION THREAD

**Discussion by fessor10** [expo/expo#18672] [Archived: Questions] [answered]
Hi there, please who can point me to the modal used in the expo go mobile application.

Is it a package? If yes, is it open source?

**Comment by raajnadar** [comment, marked as answer]
The source code of expo go is here https://github.com/expo/expo/tree/main/home

They are using reanimated-bottom-sheet

https://github.com/expo/expo/blob/main/home/menu/DevMenuBottomSheet.tsx
